### PR TITLE
yeet client configs

### PR DESCRIPTION
**Pull request by MuteTiefling**
Removes all client configs from the pack. probably snuck in over time. I don't think there's a reason to include any of these by default?

### CODE REVIEW

File: config/bhmenu-client.toml
Comment (NielsPilgaard) [inline config/bhmenu-client.toml:1]: This is needed.

File: config/fpsreducer/fpsreducer-client.toml
Comment (NielsPilgaard) [inline config/fpsreducer/fpsreducer-client.toml:1]: This is needed

File: config/itlt-client.toml
Comment (NielsPilgaard) [inline config/itlt-client.toml:1]: This is needed

File: config/patchouli-client.toml
Comment (NielsPilgaard) [inline config/patchouli-client.toml:1]: This is needed

**Review by NielsPilgaard — COMMENTED** [review]
So far I've found 4 that were needed - I'm not sure it's worth the hassle to go through them all to be honest. The chances of something breaking is fairly big here :/

Also, for these files to not be included in releases they'd need to be added to the config ignore list in the `settings.ps1`, and added to the .gitignore file. All in all, quite a bit of work 😅

**Comment by MuteTiefling** [comment]
Hrm

> So far I've found 4 that were needed - I'm not sure it's worth the hassle to go through them all to be honest. The chances of something breaking is fairly big here :/
> 
> Also, for these files to not be included in releases they'd need to be added to the config ignore list in the `settings.ps1`, and added to the .gitignore file. All in all, quite a bit of work 😅

Hrm. Alright maybe we just close this out then and yeet individual configs if/when they become an issue.

**Comment by NielsPilgaard** [comment]
Yes please ❤️

**Comment by theboo** [comment]
in the past, havent we shipped *needed* client configs in default configs folder? or am i just confused as normal

**Comment by NielsPilgaard** [comment]
> in the past, havent we shipped _needed_ client configs in default configs folder? or am i just confused as normal

If we think users can benefit from editing them, yes, if we want to "hardcode" them they go in the config folder.

**Pull request closed without merging**